common/src/Microsoft.Azure.IIoT.Core/src/Hub/Services/IoTHubEdgeBaseDeployment.cs

```diff
@@ -96,6 +125,19 @@ public sealed class IoTHubEdgeBaseDeployment : IHostProcess {
                 ""timeToLiveSecs"": 7200
             }
         }
+    },
+    ""metricscollector"": {
+        ""properties.desired"": {
+            ""schemaVersion"": ""1.0"",
+            ""scrapeFrequencySecs"": 300,
+            ""metricsFormat"": ""Json"",
+            ""syncTarget"": ""AzureLogAnalytics"",
+            ""endpoints"": {
```

Comment:
This is a problem for layered deployments as we would need to know the other modules ahead of time.  I would suggest at least having its own metrics layered deployment class we can side deploy with all other layered deployments.